File: Data Structures & Algorithms/duplicate-integer/submission-2.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Arrays.sort(nums);
                int k = 1;
        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] == nums[k]) {
                return true;
            }
            k++;
        }
        return false;
    }
}

File: Data Structures & Algorithms/is-subsequence/submission-2.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        if len(s) == 0:
            return True

        l = 0
        for c in t:
            if l == len(s):
                return True
            if c == s[l]:
                l += 1


        return l == len(s)